<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attenuator</class>
 <widget class="QDialog" name="Attenuator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attenuator Synthesis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxTopology">
       <property name="title">
        <string>Topology</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboTopology">
          <item>
           <property name="text">
            <string>Pi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tee</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bridged Tee</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTopologyImage">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxInput">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelAttenuation">
            <property name="text">
             <string>Attenuation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinAttenuation">
            <property name="suffix">
             <string> dB</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelZin">
            <property name="text">
             <string>Zin</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinZin">
            <property name="suffix">
             <string> Ohm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelZout">
            <property name="text">
             <string>Zout</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinZout">
            <property name="suffix">
             <string> Ohm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxOutput">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelR1">
            <property name="text">
             <string>R1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinR1">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="specialValueText">
             <string>--</string>
            </property>
            <property name="suffix">
             <string> Ohm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelR2">
            <property name="text">
             <string>R2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinR2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="specialValueText">
             <string>--</string>
            </property>
            <property name="suffix">
             <string> Ohm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelR3">
            <property name="text">
             <string>R3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinR3">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="specialValueText">
             <string>--</string>
            </property>
            <property name="suffix">
             <string> Ohm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusBar">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Attenuator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Attenuator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
